.nvmrc

```diff
@@ -1 +0,0 @@
-12.1
```

Comment:
this file should not be deleted or emptied. It is not solely used for the dev container